### ISSUE TITLE
Narrator + Edge doesn't read the description specified by subtitleAriaId when opening the Dialog

### ISSUE THREAD

**Issue by YusongLiu** [OfficeDev/office-ui-fabric-react#2491]
### Bug Report

- __Package version(s)__: 4.30.0
- __Browser and OS versions__: Edge latest version

#### Describe the issue:

The setting of the Dialog I used:

```
<Dialog
  modalProps={ {
    subtitleAriaId: ...
  } }
  ...
>
  <p id={ ... }>{ ... }</p>
</Dialog>
```

#### Actual behavior:

Narrator + Edge doesn't read the description specified by subtitleAriaId when opening the Dialog.

#### Expected behavior:

Narrator + Edge should read the description specified by subtitleAriaId when opening the Dialog.

**Comment by mdahamiwal** [comment]
Works fine for me with narrator + Edge. @YusongLiu, it would help if there is a codepen here.

**Comment by YusongLiu** [comment]
@mdahamiwal Here is a codepen:
https://codepen.io/anon/pen/zdLLVP

I still cannot get the subtitle being read by narrator + Edge.

**Comment by YusongLiu** [comment]
I found it's the problem of my win10 system. Tested in another PC and there is no issue anymore. Thanks!

**Comment by zelliott** [comment]
This is still an issue for me with Windows Server 2016 and Internet Explorer 11, latest version.  Only the titleAriaId and the focused button are read by Narrator.
